.github/workflows/release.yml

```diff
@@ -150,10 +151,12 @@ jobs:
       - name: Import GPG key
         if: ${{ steps.dist.outputs.support == 'yes' && steps.dist.outputs.ignore == 'no' }}
         id: import_gpg
-        uses: crazy-max/ghaction-import-gpg@v5
```

Comment:
I've removed this because it's annoying to use. We have problem with it before because the way it wants the private key to be provided is somewhat peculiar. I've changed it to just use a simple `gpg --import`. Less moving parts this way anyway.